ui/shared/components/StyledComponents.jsx

```diff
@@ -3,22 +3,24 @@ import styled from 'styled-components'
 import { Button, Header, Icon, Label, Table } from 'semantic-ui-react'
 import { NavLink } from 'react-router-dom'
 
-const BaseButtonLink = styled(({ color, padding, ...props }) => <Button {...props} />).attrs(
+const BaseButtonLink = styled(({ color, padding, background, ...props }) => <Button {...props} />).attrs(
```

Comment:
I'm a little confused why you would want to assign something specifically designed to look like an html link a background. It strikes me that a link with a background color would not look great or be particularly readable. I think maybe an icon next to the link in the color, perhaps with a hover text to explain the color/ more details would be better. Or if you really want a background, it would be better to use an actual button and not a link

Reply:
Additional padding was added to make the background look more like a button.  The intent is to have this look like a link button but still somewhat consistent with what normal non-pagenes look like.  Attached are screenshots for both.

<img width="1166" alt="Screen Shot 2021-09-11 at 13 19 19" src="https://user-images.githubusercontent.com/117335/132934661-3c751694-b049-4d91-80f4-568e62b242eb.png">
<img width="1172" alt="Screen Shot 2021-09-11 at 13 19 06" src="https://user-images.githubusercontent.com/117335/132934657-6ee7e5c5-9f9a-453e-9c3d-9fd34aefd1fe.png">

Note in second screenshot, it was taken with a mouse hover over gene "CACNB4" so it's more like a link button.

In a future PR, we may well include additional details with some kind of hover as you suggested.  However, those details still needs some finalising.  Is it OK to include confidence_level/colour for now?  Users of PanelApp are well aware of what the colours mean already and the colours add most value for now.

Reply:
this really seems like a misuse of the component, I'm sorry but I'm not willing to accept this change. It looks weird to have the blue link text on a colored background! I would say either we need to go with a different component if there is a pagene, either adding an icon next to it or using a regular basic button, or we could go with changing the link color itself, which is already supported and will look fine

Reply:
Please see reply from @david-ma below.